cluster/restore-from-backup.sh

```diff
@@ -185,15 +190,13 @@ elif [ "${ETCD_API}" == "etcd3" ]; then
 
   # Run etcdctl snapshot restore command and wait until it is finished.
   # setting with --name in the etcd manifest file and then it seems to work.
-  # TODO(jsz): This command may not work in case of HA.
-  image=$(docker run -d -v ${BACKUP_DIR}:/var/tmp/backup --env ETCDCTL_API=3 \
+  docker run -v ${BACKUP_DIR}:/var/tmp/backup --env ETCDCTL_API=3 \
     "gcr.io/google_containers/etcd:${ETCD_VERSION}" /bin/sh -c \
-    "/usr/local/bin/etcdctl snapshot restore ${BACKUP_DIR}/${snapshot} --name ${NAME} --initial-cluster ${NAME}=http://localhost:2380; mv /${NAME}.etcd/member /var/tmp/backup/")
+    "/usr/local/bin/etcdctl snapshot restore ${BACKUP_DIR}/${snapshot} --name ${NAME} --initial-cluster ${INITIAL_CLUSTER} --initial-advertise-peer-urls ${INITIAL_ADVERTISE_PEER_URLS}; mv /${NAME}.etcd/member /var/tmp/backup/"
   if [ "$?" -ne "0" ]; then
     echo "Docker container didn't started correctly"
     exit 1
   fi
-  echo "Prepare container exit code: $(docker wait ${image})"
```

Comment:
Why do you remove this one?

Reply:
I removed '-d' from docker run so that stdout/stderr goes directly to console. It's a way easier to see if it worked and debug if it failed.

So 'docker wait' is not longer needed (if command fails, the if above will catch it)